### DISCUSSION TITLE
Image widget must allow for responsive versions.

### DISCUSSION THREAD

**Discussion by samuelgoldenbaum** [elementor/elementor#22136] [Feature Request]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
It is absolutely essential to be able to serve responsive images, in addition to responsive background images. In today's mobile-first world, sites cannot serve desktop media which is larger, and may include varying art direction considerations, to mobile users. 

**Describe the solution you'd like**
Allow responsive versions for images.

**Describe alternatives you've considered**
Responsive background images are not a solution and both are needed.

**Additional context**
Developers are simply closing tickets that refer to this saying images should simply scale. This is incorrect and further evidenced by Elementor's use of responsive background images. Furthermore, requests like that should not closed, but kept open for discussion.

**Comment by nicholaszein** [comment]
Hi there,

### ✈️ This Feature Request has been migrated from issue #14358. 

if you still think this request should be considered for our roadmap, please cast your **⬆️ Upvote** on the first comment.

Happy voting! 🙂

**Comment by brent-e** [comment]
Please do this, it would be far more efficient than the current kludgy workaround of adding multiple images for multiple devices. Especially given that it's already possible to do this with background images.
